src/components.ts

```diff
@@ -47,7 +53,7 @@ export async function initComponents(): Promise<AppComponents> {
   const tracing = await createTracingComponent({ config, logs })
 
   const httpServer = await createServerComponent<GlobalContext>(
-    { config, logs },
+    { config: apiSeverConfig, logs },
     {
       cors: {
         methods: ['GET', 'HEAD', 'OPTIONS', 'DELETE', 'POST', 'PUT'],
```

Comment:
Not related but needs to be done haha
```suggestion
        methods: ['GET', 'HEAD', 'OPTIONS', 'DELETE', 'POST', 'PUT', 'PATCH'],
```